### ISSUE TITLE
Cannot `convert` an object of type Error when using StandardScaler

### ISSUE THREAD

**Issue by vinson2233** [ChifiSource/Lathe.jl#63]
Julia version : v"1.5.4"
Lathe version : "0.1.6"

Minimal code to reproduce the error :
```
using Lathe.preprocess: StandardScaler
using CSV, DataFrames
csv_file = CSV.File(download("https://gist.githubusercontent.com/michhar/2dfd2de0d4f8727f873422c5d959fff5/raw/fa71405126017e6a37bea592440b4bee94bf7b9e/titanic.csv")) # Read the data, use download() if the data from internet
df = DataFrame(csv_file)
StandardScaler(df[!,:Fare]) #df[!,:Fare] is 891-element Array{Float64,1}
```

Errors : 
```
MethodError: Cannot `convert` an object of type Lathe.stats.var"#5#9"{Complex{Float64},Float64,Lathe.stats.var"#apply#6"{Complex{Float64},Float64},Lathe.stats.var"#cdf#8"{Complex{Float64},Float64}} to an object of type Lathe.stats.Distribution
Closest candidates are:
  convert(::Type{T}, ::T) where T at essentials.jl:171

Stacktrace:
 [1] StandardScaler(::Array{Int64,1}) at /Users/vinson.ciawandy/.julia/packages/Lathe/ZRJQf/src/preprocess/feature_scaling.jl:143
 [2] top-level scope at In[135]:1
 [3] include_string(::Function, ::Module, ::String, ::String) at ./loading.jl:1091
```

I have read the documentation and can't figure out what's wrong with my code
![image](https://user-images.githubusercontent.com/33550590/111867707-5cd66e80-89a8-11eb-91aa-0f71abc49498.png)

**Comment by emmettgb** [comment]
Hi,
I believe this was a bug present in the apply() method for the normal distribution, I believe the issue is resolved as of 0.1.7. I went ahead and tested it and threw the notebook  in the Lathe books repository, because I figured that would be a great way to show you the usage:
https://github.com/emmettgb/Lathe-Books/blob/main/preprocess/Lathe%20Standard%20Scalar%200.1.7%20test.ipynb
Let me know if that helps you out!

**Comment by vinson2233** [comment]
Alright, thanks for the answer.
It seems the latest lathe version available in Pkg is 0.1.6 ?
I don't know whether I should make a separate issue or not, but how do I install the 0.1.7 version?
Here's my attempt :
```
(@v1.5) pkg> add Lathe#0.1.7
   Updating git-repo `https://github.com/emmettgb/Lathe.jl.git`
ERROR: Did not find rev 0.1.7 in repository
```
I also try to use `add Lathe#Butterball` and it make Lathe downgrade to 0.1.4

**Comment by emmettgb** [comment]
You could always add it via URL from the Pkg REPL. It appears as though merging was blocked to Julia general because of some compatibility issues with CUDA.jl versioning. I will reopen a new pull request and no block it.
